test/suites/build/build.go

```diff
@@ -15,40 +15,44 @@
 package build
 
 import (
-	"bytes"
 	"fmt"
-	"os"
 	"path/filepath"
-
-	"github.com/onsi/gomega"
+	"strings"
 
 	"github.com/sealerio/sealer/test/testhelper/settings"
 	"github.com/sealerio/sealer/utils/exec"
 )
 
-func GetFixtures() string {
-	return filepath.Join(settings.DefaultTestEnvDir, "suites", "build", "fixtures")
+// GetBuildImageName return specific image name for sealer build test
+func GetBuildImageName() string {
+	return "docker.io/sealerio/forbuildtest:v1"
```

Comment:
@Starnop , @VinceCui , do we have a better name? this image will be used to push to our repository .

Reply:
docker.io/sealerio/build-test:v1 is OK